studies/coupling_comparison/preprocessing/internal/obs_to_sowfa.ipynb

```diff
@@ -4,24 +4,15 @@
    "cell_type": "code",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    #Note: thetav does not exist so will be set to zero
</pre><p>No longer true</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/a2e-mmc/assessment/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>